win32ss/user/winsrv/consrv/frontends/gui/conwnd.c

```diff
@@ -3,7 +3,7 @@
  * PROJECT:         ReactOS Console Server DLL
  * FILE:            win32ss/user/winsrv/consrv/frontends/gui/conwnd.c
  * PURPOSE:         GUI Console Window Class
- * PROGRAMMERS:     G� van Geldorp
+ * PROGRAMMERS:     Gé van Geldorp
```

Comment:
Was this changed due to UTF8? (if so, it's OK)

Reply:
Yes.